network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/settings/geoswitching/NetpEgressServersProvider.kt

```diff
@@ -63,34 +65,8 @@ class RealNetpEgressServersProvider @Inject constructor(
             ServerLocation(
                 countryCode = it.countryCode,
                 countryName = it.countryName,
-                cities = it.cities,
+                cities = it.cities.sorted(),
```

Comment:
I moved the sorting to when the serverlocations are being retrieved instead of when it is stored as I am not sure if the sorting is preserved when inserting in the room db. Thoughts?

Reply:
Supposedly, room DB doesn't guarantee that things are return in the same order in which they were inserted. So, this seems to be safer. And since we are talking about a small list of items, I think your approach is fine.

Reply:
nit: @karlenDimla, the sort logic seems to be logic that should rather be in the UI layer, not in the domain layer.
Sorting the cities like that is fine, but this is not really about room, it's just about SQL. If you want a certain ordering to be returned it's best to put that in the SQL query.